assets/scss/modules/_carousel.scss

```diff
@@ -2,9 +2,193 @@
 /// This is a poster comment.
 /// It will apply annotations to all items from file.
 /// @access private
-/// @author
+/// @author Ornella Cicalello
 /// Classes for the Carousel component
 ////
 
 .dp-library {
+  .dp-carousel {
+    position: relative;
+    padding-bottom: $dp-spaces--lv5;
+
+    &-wrapper {
+      width: 100%;
+      height: auto;
+      border: 2px solid $dp-color-silver;
+      background: $dp-color-ghostwhite;
+      border-radius: 3px;
+      overflow: hidden;
+    }
+
+    &-content {
+      -webkit-display: inline-flex;
+      -moz-display: inline-flex;
+      -ms-display: inline-flex;
+      -o-display: inline-flex;
```

Comment:
Remember that all this does not have to be

Reply:
Yes, sorry, I'm deleting it